package.json

```diff
@@ -18,6 +18,7 @@
     "nuxt-icons": "^3.2.1",
     "prismjs": "^1.29.0",
     "rosetta": "^1.1.0",
+    "typescript-imgix-url-params": "^0.3.1",
```

Comment:
Can we move this into dev dependencies? It's not used runtime right?

Reply:
Looks like we can! I expected Nuxt build to fail if the dependency is missing but apparently not. Guess it strips out all types before building.